### PR TITLE
Add a CDK speedrun tutorial doc

### PR DESCRIPTION
**Pull request by avaidyanatha**
## Main Inspiration
We got a comment that our CDK tutorial and docs were really heavy. To write a quality connector, it is more or less required to read through a lot of the docs, but I thought I'd take a first shot at throwing an extremely barebones version of the tutorial out. This can help us iterate on a nice medium that is more consumable for a prospective connector author.

## Main Changes
- Adds speedrun doc, a stripped version of the http-source tutorial

## Notes
- This PR does not add it to our documentation site yet

### CODE REVIEW

File: airbyte-cdk/python/docs/tutorials/cdk-tutorial-any-percent/cdk-speedrun.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
+# CDK Speedrun (Any% Route)
+
```
Comment (sherifnada) [inline airbyte-cdk/python/docs/tutorials/cdk-tutorial-any-percent/cdk-speedrun.md:2]:
```suggestion
This is a blazing fast guide to building an HTTP source connector. Think of it as the TL;DR version of [../../cdk-tutorial-python-http.md](url)
```

File: airbyte-cdk/python/docs/tutorials/cdk-tutorial-any-percent/cdk-speedrun.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+> python main.py check --config sample_files/config.json
+{"type": "CONNECTION_STATUS", "connectionStatus": {"status": "SUCCEEDED"}}
+
+> python main.py check --config sample_files/invalid_config.json
+{"type": "CONNECTION_STATUS", "connectionStatus": {"status": "FAILED", "message": "Input currency BTC is invalid. Please input one of the following currencies: {'DKK', 'USD', 'CZK', 'BGN', 'JPY'}"}}
+```
+
+## Define your Stream
+
+In your `source.py` file, add this `PythonHttpExampleStream` class. This stream represents an endpoint you want to hit.
```
Comment (sherifnada) [inline airbyte-cdk/python/docs/tutorials/cdk-tutorial-any-percent/cdk-speedrun.md:117]: This is not the name of the class
```suggestion
In your `source.py` file, add this `ExchangeRates` class. This stream represents an endpoint you want to hit.
```

**Review by sherifnada — APPROVED** [review]

File: airbyte-cdk/python/docs/tutorials/cdk-tutorial-any-percent/cdk-speedrun.md (hunk 2 above)
Comment (sherifnada) [reply airbyte-cdk/python/docs/tutorials/cdk-tutorial-any-percent/cdk-speedrun.md:117]: There's a couple of references to this throughout so you might wanna double check. I just did a speedrun review

File: airbyte-cdk/python/docs/tutorials/cdk-tutorial-any-percent/cdk-speedrun.md (hunk 1 above)
Comment (avaidyanatha) [reply airbyte-cdk/python/docs/tutorials/cdk-tutorial-any-percent/cdk-speedrun.md:2]: Done!

**Pull request merged**